magwest/templates/guest_checklist/stage_plot_deadline.html

```diff
@@ -0,0 +1,33 @@
+{% extends "guest_checklist/stage_plot_deadline.html" %}
```

Comment:
I don't think you can have a file extend itself. Did you mean to name this file `band_stage_plot_deadline.html`?

Reply:
The following files all extend themselves; I assume this is extending from the parent plugin it would otherwise be overriding:
guest_checklist/band_merch_deadline.html
guest_checklist/mc_deadline.html
guests/band_agreement.html

Reply:
Oh neat! I didn't realize we could do that. In that case, this looks good!